rbac/common/manager/rbac_manager.py

```diff
@@ -13,11 +13,90 @@
 # limitations under the License.
 # -----------------------------------------------------------------------------
 
+import logging
+import inspect
+
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+from rbac.common.protobuf.rbac_payload_pb2 import RBACPayload
 from rbac.common.user.user_manager import UserManager
 from rbac.common.role.role_manager import RoleManager
+from rbac.common.task.task_manager import TaskManager
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _only_rbac_common_(value):
```

Comment:
Can you add docstrings to these functions/classes?

Reply:
resolved